charts/flyte-core/templates/flytescheduler/configmap.yaml

```diff
@@ -1,3 +1,4 @@
+{{- if .Values.flytescheduler.enabled}}
 {{- if .Values.workflow_scheduler.enabled}}
   {{- if eq .Values.workflow_scheduler.type "native" }}
```

Comment:
Not your current change but we have flytescheduler and workflow_scheduler both here. 
Can we just keep flytescheduler .
Similarly instead of workflow_scheduler.type we should have flytescheduler.type

Reply:
Created a different PR for that https://github.com/flyteorg/flyte/pull/1896